pyproject.toml

```diff
@@ -10,8 +10,8 @@ python = ">=3.8, <3.11"
 tensorflow = {version = "2.8.0", markers = "platform_machine  == 'x86_64'"}
 tensorflow-macos = {version = "2.8.0", markers = "platform_machine  == 'arm64'"}
 onnx = {version = "^1.11.0", markers = "platform_machine  == 'x86_64'"}
-protobuf = {version = "^3.20.1", markers = "platform_machine  == 'x86_64'"}
-keras-data-format-converter = "0.0.17"
+protobuf = "3.20.^0"
```

Comment:
The version number looks odd is that intentional? and why did we decide to install it always instead of only for x86?